tests/e2e/default/targets/target-accuracy.wdio-spec.js

```diff
@@ -170,4 +172,23 @@ describe('Target accuracy', () => {
     const docs = await utils.db.allDocs({ startkey: 'target', endkey: 'target\ufff0' });
     expect(docs.rows.length).to.equal(1);
   });
+
+  it('should handle old format of the rules-state-store', async () => {
+    const docId = '_local/rulesStateStore';
+    const localRulesStateStore = await chtDbUtils.getDoc(docId);
+    await chtDbUtils.updateDoc('_local/rulesStateStore', {
+      rulesStateStore: {
+        ...localRulesStateStore.rulesStateStore,
+        targetState: localRulesStateStore.rulesStateStore.targetState.targets
+      }
+    });
+
+    await commonPage.refresh();
+    await analyticsPage.goToTargets();
+    await commonPage.waitForPageLoaded();
+
+    const targets = await analyticsPage.getTargets();
+    console.log(targets);
```

Comment:
Remove this?

Reply:
yea, I've added an additional check to this test :) it's pending the push. I'm doing one last manual upgrade test locally now, and then requesting a review. 
I am happy you're around :)

Reply:
I'm always watching...